sonarr4kinstall.sh

```diff
@@ -63,16 +72,28 @@ fi
 
 echo_progress_start "Generating configuration"
 
+# Start sonarr to config
+systemctl stop sonarr.service >>$log 2>&1
+
 cat > /home/${user}/.config/sonarr4k/config.xml << EOSC
 <Config>
+  <LogLevel>info</LogLevel>
+  <UpdateMechanism>BuiltIn</UpdateMechanism>
+  <Branch>develop</Branch>
+  <BindAddress>127.0.0.1</BindAddress>
   <Port>8882</Port>
   <SslPort>9898</SslPort>
-  <UrlBase>/sonarr4k</UrlBase>
-  <BindAddress>127.0.0.1</BindAddress>
-  <UpdateMechanism>none</UpdateMechanism>
+  <EnableSsl>False</EnableSsl>
+  <LaunchBrowser>False</LaunchBrowser>
+  <AuthenticationMethod>None</AuthenticationMethod>
+  <UrlBase>sonarr4k</UrlBase>
+  <UpdateAutomatically>False</UpdateAutomatically>
 </Config>
 EOSC
-chown -R ${user}:${user}/home/${user}/.config/sonarr4k/config.xml
+
+chown -R ${user}:${user} \/home/${user}/.config/sonarr4k/
+systemctl enable --now sonarr.service >>$log 2>&1
```

Comment:
Why are we starting sonarr?

Reply:
to start the stopped instance from line 76